.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+python:
+    - "2.7"
```

Comment:
Would you add support for Python 3.5 so we can use this for newer IDAs (e.g. course-discovery)?